CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ Classify the change according to the following categories:
 ## Develop
 ### Fixed
 - Convert `max_electric_load_kw` to _Float64_ before passing to function `get_chp_defaults_prime_mover_size_class`
+- Increased the big-M bound on maximum net metering benefit to prevent artificially low export benefits
```

Comment:
Is this missing the fix to the max energy and demand tiers?

Reply:
yes, thanks for catching!  this is amended in https://github.com/NREL/REopt.jl/commit/209ba633c5601b4fdb0d417e9019e6616ebb42b8